packages/core/src/NoneTool.ts

```diff
@@ -0,0 +1,75 @@
+import path from 'path';
+import fs from 'fs-extra';
+
+import { Tool, ToolType, Package, PackageJSON, Packages, InvalidMonorepoError } from './Tool';
+import { expandPackageGlobs } from "./expandPackageGlobs";
+
+export const NoneTool : Tool = {
```

Comment:
```suggestion
export const NoneTool: Tool = {
```

Reply:
In https://github.com/Thinkmill/manypkg/pull/152 I'm bumping Prettier and all - my VS code recognizes it there and uses it to format files on save but perhaps we are missing a precommit hook or something that would help us to keep this clean?

Reply:
Most of this is probably my fault -- once I sandpaper some rough edges expect all my code to match whatever standard you already have.